Client/src/style/home.scss

```diff
@@ -0,0 +1,87 @@
+@import 'colors.scss';
+.home{
+    background-image: url('../imgs//theme/Background.png');
+    background-position: center;
```

Comment:
Nie obowiązkowe: można to wszystko codać do background jako całość

Reply:
Rozumiem że chodzi o wyekstraktowanie zawartości dotyczącej tła klasy home jako osobną klase background?

Reply:
nie zrobić to jako jeden parametr np. background: url('../imgs//theme/Background.png') center no-repeat;

Reply:
Rozumiem, zostanie to zrobione.